.github/workflows/ci_contracts.yml

```diff
@@ -0,0 +1,26 @@
+name: contracts
+
+on: [push, pull_request]
+
+jobs:
+  CI:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 10
```

Comment:
Do we still need to be on node 10 because we're on truffle 4?

Reply:
Actually not anymore :D